fundamentals/middleware/rate-limit/WebRate2/Program.cs

```diff
@@ -1,4 +1,4 @@
-#define SECOND // FIRST SECOND
+#define FIRST // FIRST SECOND
```

Comment:
Is it intentional for this to be part of this PR?

Reply:
Nope but it doesn't hurt anything